openproblems/tasks/_batch_integration/batch_integration_graph/methods/fastmnn_embed.py

```diff
@@ -36,7 +36,7 @@ def fastmnn_embed_full_unscaled(adata):
 
```

Comment:
maybe just put "NA" for the paper name and code url should link to the github page though